mne/time_frequency/psd.py

```diff
@@ -133,16 +238,30 @@ def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf, tmin=None,
         Min time of interest
     tmax : float | None
         Max time of interest
-    n_fft : int
+    proj : bool
+        Apply SSP projection vectors.
+    method : 'welch' | 'multitaper'
+        The method to use in calculating the PSD.
+    wel_n_fft : int
         The length of the tapers ie. the windows. The smaller
         it is the smoother are the PSDs. The default value is 256.
         If ``n_fft > len(epochs.times)``, it will be adjusted down to
         ``len(epochs.times)``.
-    n_overlap : int
```

Comment:
can't do this without deprecation ... 

unless there is a good reason for it. We should keep it.